eng/Versions.props

```diff
@@ -54,9 +44,17 @@
     <NUnit3TemplatesVersion>1.6.1</NUnit3TemplatesVersion>
   </PropertyGroup>
   <PropertyGroup>
+    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview5-27607-08</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreDotNetHostResolverPackageVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreDotNetHostResolverPackageVersion>
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
     <NetCoreAppTargetingPackVersion>$(MicrosoftNETCoreAppPackageVersion)</NetCoreAppTargetingPackVersion>
+    <NetStandardPatchSeparatorIndex>$([MSBuild]::Add($(MicrosoftNETCoreAppPackageVersion.IndexOf('.')), 1))</NetStandardPatchSeparatorIndex>
```

Comment:
I don't think we should do this manipulation in Versions.props. This is meant to be just the versions that are being inserted.

Reply:
@nguerrera 
The manipulation is necessary to resolve the version of the .NET Standard Targeting Pack that is being inserted: "NetStandardTargetingPackVersion"